.github/SUPPORT.md

```diff
@@ -0,0 +1,10 @@
+
+# Looking for support? We want to help
+
+Please post your question to X (#) using the Timber tag.
```

Comment:
What is this X/Timber stuff?

Reply:
I removed the SUPPORT.md file because the file seemed like a template with no real information, and we already cover the various channels through which people can contact us in the README, such as Google Groups, Zulip, etc.